USER_MANUAL.md

```diff
@@ -0,0 +1,48 @@
+# SpatialOS GDK for Unreal Test Gyms Project 
+
+#### What is a gym
+* A gym is a level containing a minimal number of actors, with simple behaviours, necessary to demonstrate some peice of functionality in the Spatial GDK for Unreal.
```

Comment:
I thought we used the uk spelling for behaviour because unity did that so it's in our docs. Either way I'll commit this as suggested for now and we can fix that later if needed.